raiden/network/transport/matrix/transport.py

```diff
@@ -187,14 +187,15 @@ def _check_and_send(self):
             self.transport._global_send_queue.join()
 
         self.log.debug("Retrying message", receiver=to_normalized_address(self.receiver))
-        status = self.transport._address_mgr.get_address_reachability(self.receiver)
-
-        if status is not AddressReachability.REACHABLE:
-            # if partner is not reachable, return
-            self.log.info(
-                "Partner not reachable. Skipping.", partner=pex(self.receiver), status=status
-            )
-            return
+        # TODO marcosmartinez7 status must be taked into account. This is commented because on some lc mediated paytments
+        # scenarios, the status is unreachable.
+        # status = self.transport._address_mgr.get_address_reachability(self.receiver)
+        # if status is not AddressReachability.REACHABLE:
+        #     # if partner is not reachable, return
+        #     self.log.info(
+        #         "Partner not reachable. Skipping.", partner=pex(self.receiver), status=status
+        #     )
+        #     return
```

Comment:
This is a never end discussion for me :P 

Personally i like to keep this here because we know exaclty where the code was placed and we have the TODO to keep track of this

But we can remove the code, create an issue for that and assign that issue a commit hash in order to know where the code was placed.

Just removing the code doesnt seems a good a idea for me.

Reply:
i vote for the 2nd option. so far i have not heard of convincing reasons to keep commented code around.

we can just document the commented code in the issue and keep the code base cleaner.

Reply:
Agree and done!